src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1237,43 +1228,8 @@ internal virtual void UpdateBackgroundExtraData()
             {
                 ApplyBorderline();
             }
-            if ((backgroundExtraDataUpdatedFlag & BackgroundExtraDataUpdatedFlag.CornerRadius) != BackgroundExtraDataUpdatedFlag.None)
-            {
-                ApplyCornerRadius();
-            }
-            backgroundExtraDataUpdatedFlag = BackgroundExtraDataUpdatedFlag.None;
-        }
 
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        internal virtual void ApplyCornerRadius()
```

Comment:
Bad news.

Their are some custom view who override this function.

Please mark as Obsolete it first, and delete it later.

```
[Obsolete("Do not use this, that is deprecated in API13."]
internal virtual void ApplyCornerRadius()
{
    Tizen.Log.Error("NUI", "ApplyCornerRadius() deprecated internally, Please don't use it.\n");
}
```

Reply:
Done.